spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -429,7 +429,7 @@
 
   # Test suite for GET /requests/:request_id/requests
   describe 'GET /requests/:request_id/requests' do
-    let(:parent_request) { create(:request, :name => "parent", :owner => "jdoe", :number_of_children => 2, :number_of_finished_children => 0, :tenant_id => tenant.id) }
+    let!(:parent_request) { create(:request, :name => "parent", :owner => "jdoe", :number_of_children => 2, :number_of_finished_children => 0, :tenant_id => tenant.id) }
```

Comment:
why the change is required? there is no other change in this spec

Reply:
This change fixed the previous spec test failure.